charts/kusk-gateway-envoyfleet/values.yaml

```diff
@@ -121,3 +124,20 @@ accesslog:
   #   path: "%REQ(X-ENVOY-ORIGINAL-PATH?:PATH)%"
   #   response_code: "%RESPONSE_CODE%"
   #   duration: "%DURATION%"
+
+# TLS Certificates and Ciphersuites configuration
+tls: []
```

Comment:
Only the `tlsSecrets` field is an array
`tls` itself is an object

Reply:
Oops, thanks!